docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -316,8 +316,11 @@ URL of the execution client's Engine JSON RPC APIs.
     ee-jwt-secret-file: "ee-jwt-secret.hex"
     ```
 
-Location of the file specifying the hex-encoded 256-bit secret key to be used for verifying and generating JSON Web
-Tokens.
+Shared secret used to authenticate [execution clients](../../Concepts/Merge.md#execution-and-consensus-clients) when
+using the Engine JSON-RPC API.
+Contents of file must be at least 32 hex-encoded bytes and not begin with `0x`.
```

Comment:
Contents must be exactly 32 hex-encoded bytes.  I think Teku can handle both with and without the 0x prefix, though without is probably more compatible with ELs.